styleguide/components/31-molecules/teasers/teaser-search-wide/_teaser-search-wide.scss

```diff
@@ -0,0 +1,104 @@
+.teaser.teaser-search-wide {
```

Comment:
No need to create a new teaser, it's just a teaser-search with extra stuff in it.
Less components, less duplicate sass...